### DISCUSSION TITLE
Group Layer

### DISCUSSION THREAD

**Discussion by howidoo** [RomanKubiak/ctrlr#346] [Q&A] [answered]
Is it possible to send an object in front of a group but not in the group.
So when i use tabs for instance one group stays alway on top. 
When I use component is not part of the group it jumps around and when i tray to trag it in the right position again its member of the group again :)
I knew i can solve this by using a diffrent group layout but my design is always finished and i dont want to mess around with my layout anymore. Thank for help

**Comment by dnaldoog** [comment, marked as answer]
It is much better to use layers to achieve that sort of thing.

Go to **panel-> add layer** then right click on the component (you'll have to hide the new layer if it is on top) and click **send to layer**. Make the layer visible again and you will see your component there unaffected by the group sitting in the layer below.

https://ctrlr.org/forums/topic/using-layers-tutorial-1-0/

**Reply by howidoo** [reply]
thank you. It now works with the group layer
